python/ray/serve/model_wrappers.py

```diff
@@ -248,6 +249,9 @@ async def predict(self, inp):
         """Perform inference directly without HTTP."""
         return await self.predict_impl(inp)
 
+    async def reconfigure(self, checkpoint: Union[Checkpoint, str]) -> "BatchPredictor":
+        """Returns a new Batch Predictor from Checkpoint"""
+        return await BatchPredictor.from_checkpoint(checkpoint, BatchPredictor)
```

Comment:
This should not be BatchPredictor, but rather the predictor_cls that’s returned by load_predictor_cls in init.

Reply:
@amogkam Please take a look again when you get a chance.